modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/lineartransform/v0.1.0.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/lineartransform/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "import": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0",
```

Comment:
this schema {{base}}/schemas/neurosciencegraph/atlas/transform/v0.1.0  is equivalent to {{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0 for now. I still think the entity schema should be imported and extended here.